src/App.js

```diff
@@ -11,38 +9,25 @@ import Emulator from './Emulator/Emulator';
 
 const data = {
   "id": 1,
-  "module_number": 1,
-  "module_name": "Local Area Networks",
+  "submodule_number": 1,
+  "submodule_name": "Local Area Networks",
   "objective": "Connect two hosts together and check they can communicate",
   "tasks": [
     "Attach h1 to s1.",
     "Attach h2 to s1.",
     "Launch.",
-    "In Console, run h4> ping 172.16.0.100.",   
+    "In Console, run h1> arp 172.16.0.100.",
+    "In Console, run h1> ping 172.16.0.100.",
+    "In Console, run h1> arp 172.16.0.100.",
   ],
 }
 
 function App() {
   return (
-    <div>
-      <header className="navHeader">
-        <Navbar expand="lg" class="d-flex flex-column flex-md-row align-items-center px-3 pt-2 px-md-4 mb-3 bg-white border-bottom shadow-sm">
-        <Navbar.Brand href="#home">
-            <img src={logo} class= "logo"/>
-        </Navbar.Brand>
-        <Navbar.Toggle aria-controls="basic-navbar-nav" />
-        <Navbar.Collapse id="basic-navbar-nav">
-            <Nav className="ml-auto">
-                <Nav.Link href="#home">Home</Nav.Link>
-                <Nav.Link href="#about">About</Nav.Link>
-                <Nav.Link href="#contact">Contact Us</Nav.Link>
-                <Button variant="link">Sign In</Button>
-            </Nav>
-        </Navbar.Collapse>
-        </Navbar>
-      </header>
+    <>
```

Comment:
I've never see this before since I didn't use JSX on my team, it is just grouping `Header` & `Emulator` so they can be grouped??

Reply:
Yeah so in React components, either in a class or as a function, you are only allowed to return one component. You are not allowed to return more than one component like this:

`const example = () => {
  return (
    <h1>Hello</h1>
    <h1>World!</h1>
  );
};
`

Instead, you either have to wrap everything in a div or do what I did, which is wrapping the components in what is called a JSX **fragment**. There is a subtle difference between the div solution and the fragment solution. In the div solution, an actual div will be added to the DOM, which is not the case for the fragment (guessing this is because of some Babel magic). More on this [here](https://reactjs.org/docs/fragments.html).

Reply:
I always hated the div solution which I had to do in Mithril. Awesome.